dotCMS/gradle.properties

```diff
@@ -1,6 +1,6 @@
 dotcmsReleaseVersion=master
-coreWebReleaseVersion=latest
-webComponentsReleaseVersion=latest
+coreWebReleaseVersion=next
```

Comment:
Are these web properties OK?

Reply:
yes it's part of the new naming conventions of dotcms-ui @vicozizou